_pytest/assertion/__init__.py

```diff
@@ -113,7 +115,8 @@ def callbinrepr(op, left, right):
         for new_expl in hook_result:
             if new_expl:
                 if (sum(len(p) for p in new_expl[1:]) > 80*8
-                        and item.config.option.verbose < 2):
+                        and item.config.option.verbose < 2
+                        and 'CI' not in os.environ):
```

Comment:
That was actually copied from [phpunit-vw](https://github.com/hmlb/phpunit-vw/blob/master/src/HMLB/VW/SecretSoftware.php#L20-L34) - I'm not sure if such a big list is actually needed, I'd expect them all to set `CI`. But I'll double-check.